script/brave_license_helper.py

```diff
@@ -144,4 +144,3 @@ def CheckBraveMissingLicense(target_os, path, error):
         else:
             if path in ANDROID_ONLY_PATHS:
                 return  # Android failures are not relevant on desktop.
-        raise error
```

Comment:
Did you commit that by mistake?

Reply:
@fmarier The PR is currently in draft, I will let you know once the PR is ready to review as it needs rebasing against 1.14.x and a quick pass to make sure no obvious issues, thanks

Reply:
It's in draft, mainly so that when I rebase I don't have to redo these changes, but it'll be undone once finalized.